api/pom.xml

```diff
@@ -20,6 +20,27 @@
             <version>1.16.16</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Pls delete useless dependecies from here

Reply:
Please delete this mockito not the one in service layer!